src/index.html

```diff
@@ -4,9 +4,320 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="header__logo"
+        >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="#Apple" class="nav__link is-active">
+              apple
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="#Samsung" class="nav__link">
+              samsung
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="#Смартфоны" class="nav__link">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#Ноутбуки"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="#Гаджеты" class="nav__link">
+              гаджеты
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="#Планшеты" class="nav__link">
+              планшеты
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="#Фото" class="nav__link">
+              фото
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="#Видео" class="nav__link">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div class="container__cards">
+        <div class="card container__card" data-qa="card">
```

Comment:
Better add class `page` to tag body and set other blocks to it's elements. This way easer write code for a project without unnecessary nesting.

 More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)

This time you could leave code that you have(because you have no container in `header` but for next projects with two or more blocks(header, catalog, testimonials...) use `mix` and `page` class for body(watch video).
Better name for `container` - `catalog` according to the context(use it next time).
```suggestion
    <main>
      <div class="page__catalog catalog">
        <section class="card" data-qa="card">
```